app/api/bread/views.py

```diff
@@ -0,0 +1,95 @@
+import datetime
+
+import flask_jwt_extended as jwt
+from flask import request
+from flask_restful import Resource
+
+from app import db
+from app.api import api
+from app.models import BreadOrder, BreadType, BreadOrderDate
+from .schema import OrderSchema, BreadListSchema, BreadTypeSchema
+
+order_schema = OrderSchema(many=True)
+breadlist_schema = BreadListSchema()
+bread_types_schema = BreadTypeSchema(many=True)
+
+
+@api.resource('/bread/')
+class BreadOrderListResource(Resource):
+
+    @jwt.jwt_required
+    def get(self):
+        startdate = datetime.date.today()
+        if startdate.month >= 9:
+            startdate = startdate.replace(month=9, day=1)
+        else:
+            startdate = startdate.replace(month=1, day=1)
+
+        user = jwt.current_user
+        orderdates = BreadOrderDate.get_all_after(startdate)
+
+        for orderdate in orderdates:
+            order = (BreadOrder.query
+                     .filter(BreadOrder.bread_order_date == orderdate)
+                     .filter(BreadOrder.user == user).first())
+            if order is None:
+                orderdate.items = []
+            else:
+                orderdate.items = order.items
```

Comment:
Kan dit alles in 1 SQL query zonder te itereren? met wat JOINs tussen BreadOrderDate en BreadOrder?

Reply:
ik ben al zo ver: 
```SQL
SELECT * FROM bread_order_date AS bod LEFT JOIN (SELECT * FROM bread_order WHERE user_id == 1) AS bo ON bod.id == bo.date_id;
```
maar geen idee hoe dit te vertalen naar sqlalchemy

Reply:
werkt het? want het ziet er nog niet zo uiterst complex uit